rllib/models/preprocessors.py

```diff
@@ -175,6 +171,11 @@ def write(self, observation, array, offset):
         array[offset:offset + self._size] = np.array(
             observation, copy=False).ravel()
 
+    @property
+    @override(Preprocessor)
+    def observation_space(self):
+        return self._obs_space
```

Comment:
Passthrough the original space for the no-op preprocessor.

Reply:
Ah, makes sense. I guess it's fine in that case!